gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -1011,6 +1025,29 @@ _gtfc_setprop_FRAMEWORK (GstTensorFilterPrivate * priv,
   }
 
   gst_tensor_filter_get_available_framework (priv, fw_name);
+
+  /** set PROP_IS_UPDATABLE in case it was set before framework */
+  g_value_init (&val, G_TYPE_BOOLEAN);
+  g_value_set_boolean (&val, priv->is_updatable);
+  status = _gtfc_setprop_IS_UPDATABLE (priv, prop, &val);
+  if (status != 0) {
+    g_warning ("Set propery is-updatable failed with error: %d", status);
+    return status;
+  }
+  g_value_unset (&val);
```

Comment:
How about moving `g_value_unset` to between line 1032 and 1033? Same issue in line 1048.

Reply:
Sorry for the late reply. this is done.